completion.go

```diff
@@ -105,7 +105,9 @@ func (c *Client) CreateCompletion(
 		return
 	}
 
-	if request.Model == GPT3Dot5Turbo0301 || request.Model == GPT3Dot5Turbo {
+	switch request.Model {
+	default:
+	case GPT3Dot5Turbo0301, GPT3Dot5Turbo, GPT4, GPT40314, GPT432K0314, GPT432K:
```

Comment:
Are we sure this endpoint does not support GPT4-based models?

Reply:
Please check https://platform.openai.com/docs/models/model-endpoint-compatibility